Application/FileConverter/Services/ConversionService.cs

```diff
@@ -118,13 +119,27 @@ private void ConvertFiles()
                             }
                         }
 
+                        if (!files.Contains(conversionJob.OutputFilePath))
+                        {
+                            files.Add(conversionJob.OutputFilePath);
+                        }
+
                         break;
                     }
                 }
 
                 Thread.Sleep(50);
             }
 
+            ISettingsService settingsService = SimpleIoc.Default.GetInstance<ISettingsService>();
```

Comment:
I made some changes removing deprecated dependencies. You can now use `this.settingsService` to access the settings.